Documentation/project-docs/dogfooding.md

```diff
@@ -236,4 +236,22 @@ need to modify your .csproj to target netcoreapp2.0 and reference the nightly bu
   </PropertyGroup>
 ```
 
-In a future update to Visual Studio, it will no longer be necessary to make this edit.
\ No newline at end of file
+In a future update to Visual Studio, it will no longer be necessary to make this edit.
+
+## Finding specific builds
+
+The URL scheme for the runtime is as follows:
+
+```
+https://dotnetcli.blob.core.windows.net/dotnet/master/Installers/$version$/dotnet-win-x64.$version$.exe
+https://dotnetcli.blob.core.windows.net/dotnet/master/Installers/2.0.0-preview1-001915-00/dotnet-win-x64.2.0.0-preview1-001915-00.exe
+```
+
+The URL scheme for the SDK & CLI is as follows:
+
+```
+https://dotnetcli.blob.core.windows.net/dotnet/Sdk/$version$/dotnet-dev-win-x86.$version$.exe
+https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.0.0-preview1-005791/dotnet-dev-win-x86.2.0.0-preview1-005791.exe
```

Comment:
`https://dotnetcli.azureedge.net/...` is better since that is behind the CDN (_much_ faster from outside US).